app/views/users/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="review-list">
+  <% @user.reviews.each do |review| %>
+  <div id="book-<%= review.book.id %>-info">
+    <img src="<%= review.book.thumbnail %>" alt="Cover of <%= review.book.title %>">
+    <br><h3><%= review.book.title %></h3><br>
+    <!-- possible format change for readability -->
+  </div>
+  <div id="review-<%= review.id %>-info">
+    <h3><%= review.title %></h3>
+    <p>Written by: <%= @user.username %></p>
+    <p>Written on: <%= Date.strptime(review.updated_at.to_s)%></p>
+    <p>Rating: <%= review.rating %>/5</p>
+    <p><%= review.text %></p>
+  </div>
+  <% end %>
+  <% if @user.reviews.count == 0 %>
```

Comment:
```suggestion
  <% if @user.review_count == 0 %>
```

Reply:
Calling `reviews.count` violates MVC because `.count` is a class method :-(. So we'll have to add a class method `review_count` to our User class (which will just be `reviews.count` & the associated test(s).

Reply:
Noted, fixed on branch